app/javascript/gobierto_visualizations/webapp/visualizations/treeMapNested.vue

```diff
@@ -82,6 +82,7 @@ export default {
       updateData: false,
       dataForTableTooltip: undefined,
       dataNewValues: undefined,
+      arrayValuesContractTypes: undefined,
```

Comment:
Array better initialize them as `[]`, unless you need undefined for somehow.

Reply:
Done!